### PR TITLE
KT-29398 Add "arg" postfix template

### PR DESCRIPTION
**Pull request by t-kameyama**
https://youtrack.jetbrains.com/issue/KT-29398

### CODE REVIEW

File: idea/testData/codeInsight/postfix/arg.kt.after [hunk 1]
```diff
@@ -0,0 +1,3 @@
+fun foo(s: String) {
+    (s<caret>)
```
Comment (goodwinnk) [inline idea/testData/codeInsight/postfix/arg.kt.after]: Have you investigated why caret position is not before `()`?

**Comment by goodwinnk** [comment]
Thank you for the contribution! I have relayed your commit to master as https://github.com/JetBrains/kotlin/commit/6acf3ad629329dfcf0bf4074b1ca0add2017e911. 

Please let me know if you will find out how to resolve caret issue in tests.

**Comment by t-kameyama** [comment]
I think 0e6cf05 resolves caret issue.

**Comment by goodwinnk** [comment]
I have added it as https://github.com/JetBrains/kotlin/commit/0acecb493651f97bce1c2a68105b0d8aad0a59e5. Thanks!

**Pull request closed without merging**